security.rst

```diff
@@ -438,6 +438,9 @@ start with ``/admin``, you can:
                 # require ROLE_ADMIN for /admin*
                 - { path: '^/admin', roles: ROLE_ADMIN }
 
+                # or require ROLE_ADMIN and IS_AUTHENTICATED_FULLY for /admin*
+                - { path: '^/admin', roles: [IS_AUTHENTICATED_FULLY, ROLE_ADMIN] }
```

Comment:
Please update the `xml` and `php` examples below accordingly, thank you!

Reply:
I am not sure about the XML syntax - if some one could confirm it